app/views/hyrax/base/show.html.erb

```diff
@@ -2,33 +2,40 @@
 
 <%= render 'shared/citations' %>
 
-<div itemscope itemtype="http://schema.org/CreativeWork" class="row">
+<div class="row work-type">
   <div class="col-xs-12">
-    <header>
-      <%= render 'work_title', presenter: @presenter %>
-    </header>
-  </div>
-  <div class="col-sm-8">
-    <%= render 'work_description', presenter: @presenter %>
-    <%= render 'relationships', presenter: @presenter %>
-    <%= render 'metadata', presenter: @presenter %>
-  </div>
-  <div class="col-sm-4">
-    <%= render "show_actions", presenter: @presenter %>
-    <%# Use the fully qualified path to avoid RSpec error: translation missing: en.hyrax.base.show.html.erb.last_modified %>
-    <%= t('hyrax.base.show.last_modified', value: @presenter.date_modified) %>
-    <%= render 'representative_media', presenter: @presenter %>
-    <%= render 'social_media' %>
-    <%= render 'citations', presenter: @presenter %>
+      <%= render 'work_type', presenter: @presenter %>
   </div>
 </div>
 
-<div class="row">
-  <div class="col-sm-12">
-    <%= render 'items', presenter: @presenter %>
-    <%= render 'workflow_actions_widget', presenter: @presenter %>
-    <%# TODO: we may consider adding these partials in the future %>
-    <%#= render 'sharing_with', presenter: @presenter %>
-    <%#= render 'user_activity', presenter: @presenter %>
+<div itemscope itemtype="http://schema.org/CreativeWork" class="row">
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <%= render 'work_title', presenter: @presenter %>
+    </div>
+    <div class="panel-body">
+      <div class="row">
+        <div class="col-sm-3 text-center">
+          <%# Use the fully qualified path to avoid RSpec error: translation missing: en.hyrax.base.show.html.erb.last_modified %>
```

Comment:
I don't understand this comment. is it cruft?

Reply:
I think so. It's not mine. Maybe need to rebase, could have been removed after this.

Reply:
I think the comment was referring to the line that followed it, which you removed.  Can you remove the comment too?

Reply:
Ah yes, this comment goes with the current source: https://github.com/samvera/hyrax/blob/master/app/views/hyrax/base/show.html.erb#L18 totally should be removed. Will do.